scripts/delayed.js

```diff
@@ -115,11 +181,11 @@ if (
 ) {
   loadGTM();
   //loadAT();
-  coveoua(
-    'init',
-    accessToken,
-    `https://${organizationId}.analytics.org.coveo.com`,
-  );
-  coveoua('send', 'pageview');
+
+  if (window.location.pathname.indexOf('/products/family/') > 0 || window.location.pathname.indexOf('/products/sku/') > 0 || window.location.pathname.indexOf('/products/bundles/') > 0 ){
```

Comment:
This should be based on the page type instead of the URL. All product pages have `<meta name="template" content="ProductDetail">`.

```suggestion
  if (getMetadata('template') === 'ProductDetail' ){
```